src/App.tsx

```diff
@@ -1,15 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, inputAskEdit] = useState('');
+
+  const inputInfo: React.ChangeEventHandler<HTMLInputElement> = (event) => {
```

Comment:
```suggestion
  const handleInputChange: React.ChangeEventHandler<HTMLInputElement> = (event) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
Thanks